{"ray": {"sid": "d5cedb753f4e4528bf3e85e156de5520", "uid": null, "qid": "31112d1653d84dd58766ac1711a4f496", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [31112d1653d84dd58766ac1711a4f496]\nTraceback (most recent call last):\n  File \"/Users/apoorvgarg/Downloads/openfabric-ai-software-engineer/.conda/lib/python3.11/site-packages/openfabric_pysdk/app/worker.py\", line 95, in process\n    output = execution_callback_function(data, ray, self.state)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apoorvgarg/Downloads/openfabric-ai-software-engineer/main.py\", line 39, in execute\n    qa_response = answer_question(text,modelMethod=\"Phi2\")\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/apoorvgarg/Downloads/openfabric-ai-software-engineer/util.py\", line 26, in answer_question\n    outputs = model.generate(**inputs, max_length=200)\n                               ^^^^^^\nUnboundLocalError: cannot access local variable 'inputs' where it is not associated with a value\n", "created_at": "2024-04-04T21:10:34.019985"}], "status": "FAILED", "created_at": "2024-04-04T21:10:13.896422", "updated_at": "2024-04-04T21:10:34.603508", "finished": true}, "in": {"text": ["what is a convex lens ?"]}, "out": {}}